suite-native/transactions/src/components/TransactionDetail/TransactionDetailData.tsx

```diff
@@ -31,6 +36,7 @@ export const TransactionDetailData = ({ transaction }: TransactionDetailDataProp
     const transactionTargetAddresses = transaction.targets[0].addresses;
 
     const isTransactionPending = isPending(transaction);
+    const fiatIsVisible = !isTestnet(transaction.symbol);
```

Comment:
Always prefix boolean values with is, has, should etc. So in this case it would be `isFiatVisible`.

Altough I  don't really like that you're using this condition inverted in some components, as it is in others and now you're using some vague name even though it's in just one place. 

Did you consider creating custom hook to determine whether account is testnet or not which would just return true / false based on symbol / accountKey and then using it in all of the components?